### ISSUE TITLE
Feature Request: Add an optional parameter to allow uppercase letters in the optional-scope (RDT-793)

### ISSUE THREAD

**Issue by ustccw** [espressif/conventional-precommit-linter#25]
@tomassebestik Hello!
The default `conventional-precommit-linter` will report the FAIL if `<optional-scope>` has uppercase letters.

```
FAIL: (optional-scope) if used, must be written in lower case without whitespace
```

Could we add an optional parameter (something like `--ignore-case-scope`) to allow uppercase letters in the `<optional-scope>`.
Thanks.

**Comment by tomassebestik** [comment]
@ustccw Feature available in release `v1.9.0` (update your `.pre-commit-config.yaml`)

**Comment by ustccw** [comment]
@tomassebestik thanks a lot!
